person.rb

```diff
@@ -2,13 +2,14 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_accessor :age, :name
+  attr_accessor :age, :name, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
-    super()
+    super(name)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] As mentioned by the previous reviewer, we were asked to implement the `add_rental` method, let us kindly implement it